source/_includes/code/application-framework/login-page/doc.md

```diff
@@ -0,0 +1,90 @@
+<table class="pforg-code-usage-table">
+  <thead>
+    <tr>
+      <th>Selector</th>
+      <th>.login-pf-page</th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td>Applied to</td>
+    <td>page wrapper</td>
+  </tr>
+  <tr>
+    <td>Required</td>
+    <td><strong>Yes</strong></td>
+  </tr>
+  <tr>
+    <td>Summary</td>
+    <td>Adds overall styling</td>
+  </tr>
+  </tbody>
+</table>
+
+<table class="pforg-code-usage-table">
+  <thead>
+    <tr>
+      <th>Selector</th>
+      <th>.login-pf-page-accounts</th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td>Applied to</td>
+    <td><code>.login-pf-page</code> on page wrapper</td>
+  </tr>
+  <tr>
+    <td>Required</td>
+    <td><strong>No</strong></td>
+  </tr>
+  <tr>
+    <td>Summary</td>
+    <td>for login page with multiple account login options</td>
+  </tr>
+  </tbody>
+</table>
+<table class="pforg-code-usage-table">
+  <thead>
+    <tr>
+      <th>Selector</th>
+      <th>.login-pf-social-link-more</th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td>Applied to</td>
+    <td>any <code>li</code> used in the <code>ul.login-pf-social</code></td>
+  </tr>
+  <tr>
+    <td>Required</td>
+    <td><strong>No</strong></td>
+  </tr>
+  <tr>
+    <td>Summary</td>
+    <td>hides <code>li</code> in a expandable container. </td>
+  </tr>
+  </tbody>
+</table>
+
+<table class="pforg-code-usage-table">
+  <thead>
+    <tr>
+      <th>Selector</th>
+      <th>.login-pf-socal-double-col</th>
```

Comment:
I spelt social wrong :(

Reply:
Whoops, didn't notice that lol. Will fix the typo.